Testing.cs

```diff
@@ -41,16 +41,18 @@ static void Main(string[] args)
             System.Console.WriteLine("\n\tDeal one card\n");
             Pack.Deal().Currentcard();
 
-            /*System.Console.WriteLine("\n\tShow deck\n");
+            System.Console.WriteLine("\n\tShow deck\n");

```

Comment:
Good formatting for terminal output, code is also organised and grouped making it more legible.